docs/cfc.md

```diff
@@ -0,0 +1,23 @@
+# nf-core/configs: CFC Configuration
+
+All nf-core pipelines have been successfully configured for use on the CFC cluster at the insert institution here.
+
+To use, run the pipeline with `-profile cfc`. This will download and launch the [`cfc.config`](../conf/cfc.config) which has been pre-configured with a setup suitable for the CFC cluster. Using this profile, Nextflow will download a singularity image with all of the required software before execution of the pipeline.
+
+Before running the pipeline you will need to load Nextflow and Singularity using the environment module system on CFC cluster. You can do this by issuing the commands below:
+
+```bash
+## Load Nextflow and Singularity environment modules
+module purge
+module load devel/java_jdk/1.8.0u121
+module load qbic/singularity_slurm/3.0.1
+
+## Example command for nf-core/atacseq
+nextflow run nf-core/atacseq -profile PROFILE --genome GRCh37 --design /path/to/design.csv --email test.user@crick.ac.uk
```

Comment:
`-profile cfc` instead of `-profile PROFILE`

Reply:
I agree, lets remove this and refer to the respective pipeline readme/documentation